### PR TITLE
adds drones to AIC lab, ports tegu drones

### PR DESCRIPTION
**Pull request by quadbium**
## About the Pull Request
adds a drone fabricator to AIC lab, ports tegu drones.
ports: https://github.com/vlggms/tegustation-bay12/pull/357

## Why It's Good For The Game
way for the AIC to directly interact with the station, implement existing ghost roll.

## Changelog

:cl:
add: Added a drone fabricator to the AIC lab
add: Added AIC controlled drones
qol: You can now click on drones to join as them
/:cl:

### CODE REVIEW

**Comment by TerryTehCoder** [comment]
Is it a good idea to allow the AIC to circumvent remote drone shutdown commands when operating a drone chassis? Perhaps it should have a confirmation prompt on the terminal end but not entirely able to ignore it. 

This is more of an administrative style discussion than any issue with the code.

**Comment by Vizoplez** [comment]
> Is it a good idea to allow the AIC to circumvent remote drone shutdown commands when operating a drone chassis? Perhaps it should have a confirmation prompt on the terminal end but not entirely able to ignore it.
> 
> This is more of an administrative style discussion than any issue with the code.

yeah this is fine.

Clarification: The AI is almost never rogue (malf AI is super rare) and if the AI is ignoring its laws and abusing drones to act combatatively/aggressively that falls specifically under staff purview, not so much code.

File: code/modules/mob/living/silicon/robot/drone/drone.dm [hunk 1]
```diff
@@ -32,12 +32,12 @@ var/list/mob_hat_cache = list()
 	pass_flags = PASS_FLAG_TABLE
 	braintype = "Drone"
 	lawupdate = FALSE
-	density = TRUE
+	density = FALSE
 	req_access = list(ACCESS_ENGINEERING_LVL2, ACCESS_ROBOTICS)
 	integrated_light_max_bright = 0.5
 	local_transmit = 1
 	possession_candidate = 1
-	speed = -2
```
Comment (Tsurupeta) [inline code/modules/mob/living/silicon/robot/drone/drone.dm:40]: why

Comment (quadbium) [reply code/modules/mob/living/silicon/robot/drone/drone.dm:40]: that's what the Tegu code did, I think it makes the drones faster?

Comment (Tsurupeta) [reply code/modules/mob/living/silicon/robot/drone/drone.dm:40]: It does make them faster, but why should they be faster?

Comment (quadbium) [reply code/modules/mob/living/silicon/robot/drone/drone.dm:40]: "Allows AI to do more stuff, simple as that.
    Drones used to be too fast and could block people(especially annoying). This isn't the case anymore.
    Just QOL."

**Pull request merged**